js/views/pageNavVw.js

```diff
@@ -591,14 +611,175 @@ module.exports = baseVw.extend({
       Backbone.history.navigate(route, {trigger: true});
     });
   },
+  
```

Comment:
You're right, views should not access each other's DOM elements. But, views can communicate in other ways depending on the situation, namely by calling each other's methods (if you have a reference to other view). Also, views can communicate via events.

If we were using a plug-in, it would probably be fine for this to be in this view, since the plug-in would encapsulate a lot of the common functionality. To be honest, if we don't use a plugin here, whoever is tasked with implementing this on a text field in a different view, will probably need to bring a plug-in in and then refactor this code.